tests/api_tests/test_participant_summary_api.py

```diff
@@ -4424,3 +4426,59 @@ def test_updated_since_parameter(self):
             unexpected_participant.participantId in response_ids
             for unexpected_participant in unexpected_participant_list
         ))
+
+    def test_remote_identity_verified(self):
+        """ Test to see if a remote ID verification True Response saves successfully """
+        # Set up user config to have client set to vibrent
+        user_info = config.getSettingJson(config.USER_INFO)
+        original_user_client_id = user_info['example@example.com']['clientId']
+        user_info['example@example.com']['clientId'] = 'vibrent'
+        config.override_setting(config.USER_INFO, user_info)
```

Comment:
Shouldn't need this for the tests, you can do the POST with the default account.